### ISSUE TITLE
Configurable UI Functionality?

### ISSUE THREAD

**Issue by scottyeck** [Hacklone/private-bower#176]
Is there any interest in enabling some configurability of the UI? For example, one might want the ability to set a configuration option that would disable the delete button so that users can't access the registry and delete entries willy-nilly. FWIW, we're interested in this functionality.

I'm happy to do the work and submit a PR - just want to take the temperature and understand the probability of it making it in.

**Comment by amill-kft** [comment]
Hey,
If you enable authentication it doesn't allow anyone to delete packages, though I get that this ain't the same as you're talking about.

Barna

**Comment by Hacklone** [comment]
Hey,
If you enable authentication it doesn't allow anyone to delete packages, though I get that this ain't the same as you're talking about.

Barna

**Comment by scottyeck** [comment]
Hey! Yeah I'm aware - I'm using the authentication, but would love that UI configurability to add one more layer of abstraction. I have no problem with the *ability* to add/delete packages, I just would prefer if it weren't so easy, considering a registered package does, by nature, establish itself as dependency for other projects in a given stack. A package going missing would almost certainly be problematic for any user.

TLDR; Auth is great, but the harder it is to delete a package, the fewer people will attempt to do so, causing fewer problems overall.

Again, I'm not requesting a feature, per se, as I'm happy to submit a PR! :metal:

**Comment by SebastianSchenk** [comment]
+1

Additionally, it would be great to add a setting in order to remove the "Fork me on Github" image.
Besides that, the "created by" footer could be a little bit more minimalistic =)

**Comment by cdibbs** [comment]
+1

Is there any word on this? Thanks!
